src/api/route-services/projects.js

```diff
@@ -65,6 +65,20 @@ class ProjectsService {
     }
   }
 
+  async getProjects() {
+    const params = {
+      TableName: this.tableName,
+    };
+    const dynamodb = createDynamoDbInstance();
+    const response = await dynamodb.scan(params).promise();
```

Comment:
This scan will need to be over the experiments table, then with the projectIds you find there (some of them won't have projectIds so you could deal with that by adding a `FilterExpression` with `attribute_exists` in the `params`).

For loading projects based on the projectIds you got from experiments you could use BatchGetItem https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#batchGetItem-property,
also keep in mind that the experiments table has `projectId` field while projects table has `projectUuid`, so there's a decision here, either using `projectId` and changing the projects and samples tables or `projectUuid` and changing the experiments table.